book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,234 @@
+In this case you get the best of both worlds, the benefits of professional support and the reassurance of an open solution.
+You should still take regular local backups of exports from your hosting provider from which you could restore your ELN system with different hosting.
+This means that you retain the option to change providers.
+Conventionl software as a service (SaaS) platforms are highly verically integrated, the software development, hosting, and administration are package deal that you must take or leave as is.
+Using open tools breaks this up affording you the flexibility to change any of these if the need arises and you have the resources to do so, it puts you in a better bargining position.
+
+(rr-rdm-elns-choosing-automation)=
+### Automation and integration
+
+ELNs with APIs have the potential for integration with instruments that would permit direct deposition of data from connected instruments into ELNs or other connected research data management infrastructure.
+These sorts of integrations are especially useful in workflows at core facilities such as bioimaging facilities who can do things like automate a substancial part of the image metadata deposition into image data management tools such as [OMERO](https://www.openmicroscopy.org/omero/).
+
+(rr-rdm-elns-standards)=
+## ELN standards and Project governance
+
+ELNs offer many advantages over paper notebooks but they come with a considerable amount of additional complexity and a number of additional risks that it is important to have a plan to mitigate when proceeding with their adoption.
+The ELN space is still quite young and rapildy changing.
+ELNs represent a key component of digital infrastructure for the future of research.
+As such it is important to think about how the academic commuity collectively wants this infrastructure to be owned and governed in the long run at this stage in the adoption of ELNs.
+An interesting model to consider emulating here is [moodle](https://moodle.com/) an open source electronic learning environment popular with universities.
+It offers commercial support and hosting and has a network of developers across the education sector who contribute to the core project as well as creating custom integrations and extensions to meet their particular needs.
+This structure allows institutions to pool resources to fund the development of features that they all need and keeps this infrastructural code in common ownership, avoiding the worst lock-in issues arising from the use of proprietary solutions.
+This more open distributed model also allows individuals to work on their own custom extensions and integrations instead of being reliant on the ability and willingness of a company to implement a feature that you need.
+
+Many open tools have very robust export and import options, as they are not subject to the incentives trap faced by proprietary offerings, though this is far from guaranteed.
```

Comment:
It looks like this paragraph fits in better with the lock in section above?

Reply:
Addressed in: [1318565](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/1318565ac1917a33d1191552f368374fef03242a)